planner/core/testdata/integration_serial_suite_in.json

```diff
@@ -129,6 +129,18 @@
       "explain select /*+ inl_hash_join(s) */ * from t join s on t.a=s.a and t.a = s.b"
     ]
   },
+  {
+    "name": "TestPushDownAggForMPP",
```

Comment:
we can add some test cases:
- agg that follows one phase, say it's on top of a join.
- agg between two joins. They hold same join and gby keys so that need not reshuffle multi times.
- TiDB-MPP mode for group by aggregation.

Reply:
added.